packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -1,348 +1,312 @@
+      mockHttpRequest.subscribeNotifications.mockRejectedValue(mockError);
 
-      expect(result).toEqual(agentProfileMock);
+      await expect(webex.cc.register()).rejects.toThrow('Error while performing register');
+
+      expect(webex.logger.error).toHaveBeenCalledWith(
+        `file: cc: Error during register: ${mockError}`
+      );
     });
+  });
 
-    it('should resolve with success message on successful registration with $config has values', async () => {
-      webex.cc.$config = {
-        force: true,
-        isKeepAliveEnabled: false,
-        clientType: 'WebexCCSDK',
-        allowMultiLogin: true,
+  describe('stationLogin', () => {
+    it('should login successfully with LoginOption.BROWSER', async () => {
+      const options = {
+        teamId: 'teamId',
+        loginOption: LoginOption.BROWSER,
       };
-      const promise = webex.cc.register();
-
-      const agentProfileMock = {
-          teams: [
-              {
-                id: "123",
-                name: "Sandbox Team"
-              }
-          ],
-          idleCodes: [],
-          wrapUpCodes: [],
-          agentId: "123",
-          name: "test1 test2",
-          agentProfileId: "456",
-          agentMailId: "test@test.com",
-          loginVoiceOptions: [
-              "EXTENSION",
-              "BROWSER",
-              "AGENT_DN"
-          ]
-      }
-      jest.spyOn(AgentConfig.prototype, 'getAgentProfile').mockResolvedValue(agentProfileMock);
-
-      // Emit the welcome event to simulate the WebSocket message
-      eventEmitter.emit(EVENT, {
-        type: CC_EVENTS.WELCOME,
-        data: { agentId: 'mockAgentId' },
-      });
 
-      const result = await promise;
-      expect(webSocketMock.subscribeAndConnect).toHaveBeenCalledWith({
-        datachannelUrl: 'https://api.example.com/v1/notification/subscribe',
-        body: {
-          force: true,
-          isKeepAliveEnabled: false,
-          clientType: 'WebexCCSDK',
-          allowMultiLogin: true,
+      webex.cc.agentConfig = {
+        agentId: 'agentId',
+      };
+
+      const registerWebCallingLineSpy = jest.spyOn(
+        webex.cc.webCallingService,
+        'registerWebCallingLine'
+      );
+
+      const stationLoginMock = jest
+        .spyOn(webex.cc.services.agent, 'stationLogin')
+        .mockResolvedValue({} as StationLoginSuccess);
+
+      const result = await webex.cc.stationLogin(options);
+
+      expect(registerWebCallingLineSpy).toHaveBeenCalled();
+      expect(stationLoginMock).toHaveBeenCalledWith({
+        data: {
+          dialNumber: 'agentId',
+          teamId: 'teamId',
+          deviceType: LoginOption.BROWSER,
+          isExtension: false,
+          deviceId: `${WEB_RTC_PREFIX}agentId`,
+          roles: [AGENT],
+          teamName: '',
+          siteId: '',
+          usesOtherDN: false,
+          auxCodeId: '',
         },
       });
-
-      expect(result).toEqual(agentProfileMock);
+      expect(result).toEqual({});
```

Comment:
Why is the station login object empty? Shouldn't we receive LoginSuccess event with some data